File: app/src/main/java/za/co/duartbreedt/androidconceptsreference/RetrofitClient.java
package za.co.duartbreedt.androidconceptsreference;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public final class RetrofitClient {
    public static final Retrofit instance =
        new Retrofit.Builder()
            .baseUrl("https://reqres.in/api/")
            .addConverterFactory(GsonConverterFactory.create())
            .build();
}

File: app/src/main/java/za/co/duartbreedt/androidconceptsreference/services/RetrofitClient.kt
package za.co.duartbreedt.androidconceptsreference.services;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

object RetrofitClient {
    val instance: Retrofit =
        Retrofit.Builder()
            .baseUrl("https://reqres.in/api/")
            .addConverterFactory(GsonConverterFactory.create())
            .build()
}
